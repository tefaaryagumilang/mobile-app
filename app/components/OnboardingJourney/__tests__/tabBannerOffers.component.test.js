import React from 'react';
import renderer from 'react-test-renderer';
import tabBannerOffers from '../tabBannerOffers.component';

describe('tabBannerOffers component', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<tabBannerOffers />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
