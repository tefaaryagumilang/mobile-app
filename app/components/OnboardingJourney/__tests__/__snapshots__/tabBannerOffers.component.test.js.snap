// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tabBannerOffers component renders correctly 1`] = `<tabBannerOffers />`;
